adapters/tappx/tappx_test.go

```diff
@@ -38,13 +38,16 @@ func TestTsValue(t *testing.T) {
 
 	bidderTappx := bidder.(*TappxAdapter)
 
-	var test int
-	test = 0
+	var test int = 0
 	var tappxExt openrtb_ext.ExtImpTappx
 	tappxExt.Endpoint = "DUMMYENDPOINT"
 	tappxExt.TappxKey = "dummy-tappx-key"
 
 	url, err := bidderTappx.buildEndpointURL(&tappxExt, test)
+	if err != nil {
+		t.Errorf("Error in buildEndpointURL: %s", err.Error())
+		return
+	}
```

Comment:
Please use testify. This should be `require.NoError(t, err, "buildEndpointURL")`.